### ISSUE TITLE
Use Chronological Order to Display Builds

### ISSUE THREAD

**Issue by piyush286** [AdoptOpenJDK/openjdk-api-v3#116]
The ordering of builds by date need some improvement since they aren't in chronological order as shown in the screenshot below.

![image](https://user-images.githubusercontent.com/10422081/74947267-a0872f00-53c8-11ea-8b17-a09cbebdb342.png)

**Comment by johnoliver** [comment]
It seems this is because by default we sort by version first, which IMO is the correct thing to do.

The issue seems to be that the build from `2020-01-30` has a version `8.0.252+1.1.202001300436` where as `2020-02-12` is `8.0.242+8.1.202002121121`.

```
      "download_count" : 4134,
      "release_link" : "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u-2020-01-30-10-55",
      "vendor" : "adoptopenjdk",
      "release_name" : "jdk8u-2020-01-30-10-55",
      "release_type" : "ea",
      "version_data" : {
         "minor" : 0,
         "build" : 1,
         "major" : 8,
         "semver" : "8.0.252+1.1.202001300436",
         "security" : 252,
         "optional" : "202001300436",
         "openjdk_version" : "1.8.0_252-202001300436-b01",
         "adopt_build_number" : 1
      }
```
      
```  
      "download_count" : 5515,
      "release_link" : "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u-2020-02-12-20-33",
      "release_name" : "jdk8u-2020-02-12-20-33",
      "vendor" : "adoptopenjdk",
      "release_type" : "ea",
      "version_data" : {
         "adopt_build_number" : 1,
         "optional" : "202002121121",
         "openjdk_version" : "1.8.0_242-202002121121-b08",
         "security" : 242,
         "major" : 8,
         "minor" : 0,
         "build" : 8,
         "semver" : "8.0.242+8.1.202002121121"
      },
      "updated_at" : "2020-02-12T20:37:35Z"
```

So the question is why are the metadata files in `jdk8u-2020-02-12-20-33` at `242`

**Comment by johnoliver** [comment]
I see the problem, the aarch build is still at 242 and the api is picking up on that

**Comment by karianna** [comment]
@johnoliver Merged the PR - can this be closed now?

**Comment by johnoliver** [comment]
@karianna that was merged into staging, looked like it worked, #119 brings it into master
